app/controllers/api/v2/domains_controller.rb

```diff
@@ -33,6 +33,7 @@ def index
 
       api :GET, "/domains/:id/", N_("Show a domain")
       param :id, :identifier, :required => true, :desc => N_("Numerical ID or domain name")
+      param :show_hidden_parameter, :bool, :desc => N_("Should unhide hidden parameters values for display")
```

Comment:
> I think the wording is wrong, both the "should" and the "unhide", I'd say "Display hidden parameters"

Indeed, or "Display hidden parameter values" to indicate it's the value and not the record being shown.